### ISSUE TITLE
C64: Writing to color RAM 

### ISSUE THREAD

**Issue by r1me** [hbekel/xlink#3]
I can write to color RAM only with `poke` command but not `load`, regardless of memory configuration value (0x30-0x37)
I noticed that while writing to memory with `load` command, memory config value 0x33 is applied if destination memory region is in I/O area. User setting is then ignored. My suggestion to solve this issue would be to add a new boolean if user specified a memory configuration.
https://github.com/hbekel/xlink/blob/c3681d96f9d3708c17d1295e19894098732c0e0c/client.c#L657

    if(range_overlaps(data, io)  && !user_specified_mem_cfg)
      command_apply_safe_memory_and_bank(self);
    else 
      command_apply_memory_and_bank(self);

**Comment by hbekel** [comment]
You're right, the user should of course get whatever they request, assuming they know what they're doing. No reason not to give them all the rope. But on the other hand, the command is called load, and that usually means loading into RAM.

I think I'd prefer to just remove the memory option from the load command completely, and instead implement a generic write command. This way load can be simplified and just always write to RAM while write will just use whatever memory config you request.
